docs/src/install-from-source.md

```diff
@@ -0,0 +1,88 @@
+# Installing from source code
+
+This installation option is better suited for Kani developers
+who need to change the Kani source code.
+
+## Dependencies
+
+In general, the following dependencies are required.
+
+> **NOTE**: These dependencies may be installed by running the CI scripts shown
+> below and there's no need to install them separately, for their respective
+> OS.
+
+1. Cargo installed via [rustup](https://rustup.rs/)
+2. [CBMC](https://github.com/diffblue/cbmc) (>= 5.58.1)
```

Comment:
I think this is now 5.59.0, please check against the latest version.

Reply:
That seems to be correct. Latest version on Brew is 5.59.0.